afs-security-analysis-local/src/test/java/com/powsybl/afs/security/local/SecurityAnalysisMock.java

```diff
@@ -0,0 +1,26 @@
+package com.powsybl.afs.security.local;
+
+import com.google.common.collect.ImmutableList;
+import com.powsybl.security.*;
+import com.powsybl.security.results.PreContingencyResult;
+import org.mockito.Mockito;
+
+import java.util.Collections;
+import java.util.concurrent.CompletableFuture;
+
+class SecurityAnalysisMock {
+    CompletableFuture<SecurityAnalysisReport> run() {
```

Comment:
I would propose to delete this class and just write the creation code in the provider mock